server/api/layer_api.py

```diff
@@ -70,17 +37,25 @@ def get(self, map_category):
         responses:
           200:
             description: Layers
+          400:
+            description: Client error
           401:
             description: Unauthorized, credentials are invalid
+          404:
+            description: Not Found
           500:
             description: Internal Server Error
         """
         try:
             # TODO: in future add a role filter on the layers
-            if map_category is None:
-                map_category = 'UNKNOWN'
-            layer_dto = LayerService.get_layers_by_category(str.upper(map_category))
+            # if map_category is None:
+            #     map_category = 'UNKNOWN'
+            layer_dto = LayerService.get_layers_by_category(map_category)
             return layer_dto.to_primitive(), 200
+        except LayerServiceError as e:
+            return {"Error": str(e)}, 400
+        except NotFound:
+            return {"Error": "No layers found"}, 404
```

Comment:
To me 404 implies that the endpoint or an individual requested object does not exist.  To me this is more of a search case where is is valid to find nothing, but I'm open to being persuaded otherwise.. :-)